content/git/concepts/stash/stash.md

```diff
@@ -0,0 +1,80 @@
+---
+Title: 'Stash'
+Description: 'Git stash is used to save changes locally so the developer can switch branches while developing or to get a clean version of the branch they are currently working on.'
+Subjects:
+  - 'Bash/Shell'
+  - 'Developer Tools'
+Tags:
+  - 'Git'
+  - 'Version Control'
+CatalogContent:
+  - 'learn-the-command-line'
+  - 'learn-git'
+---
+
+Stash is used when a developer would like to save changes locally to their machine rather than committing them. This is often used when a developer would like to switch branches in the middle of coding but the code they are working on is not yet ready to be committed.
```

Comment:
```suggestion
The **`stash`** command is used when a developer would like to save changes locally to their machine rather than committing them. This is often used when a developer would like to switch branches in the middle of an effort but the code they are working on is not yet ready to be committed.
```